internal/condition/condition_test.go

```diff
@@ -200,6 +200,28 @@ func TestEvaluate(t *testing.T) {
 				MissingParameters: []string{"param1"},
 			},
 		},
+		{
+			name: "mix_of_missing_params_and_strict_eval",
+			condition: &openfgav1.Condition{
+				Name:       "condition1",
+				Expression: "param1 == 'ok' && param2 == 'ok'",
+				Parameters: map[string]*openfgav1.ConditionParamTypeRef{
+					"param1": {
+						TypeName: openfgav1.ConditionParamTypeRef_TYPE_NAME_STRING,
+					},
+					"param2": {
+						TypeName: openfgav1.ConditionParamTypeRef_TYPE_NAME_STRING,
+					},
+				},
+			},
+			context: map[string]interface{}{
+				"param1": "ok",
+			},
+			result: condition.EvaluationResult{
+				ConditionMet:      false, // we apply strict evaluation
```

Comment:
I'm not sure I understand the motive or point of this comment. The condition is not met not because `param2` is missing but because `true && param2 == 'ok'` is partial. You cannot evaluate it *without* `param2`.

Did we mean `param1 == 'ok' || param2 == 'ok'`?

Reply:
Not sure what the comment refers to but will remove. I agree that the intent here was probably to use `||` in the CEL. I'll keep this case and also add another one with the `||` because both test different behaviors.

Reply:
https://github.com/openfga/openfga/pull/1606/files

yes, I think I made a mistake - the condition should have an `||`